plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/v1/GlueMetastoreModule.java

```diff
@@ -76,6 +84,10 @@ protected void setup(Binder binder)
 
         newOptionalBinder(binder, GlueColumnStatisticsProviderFactory.class)
                 .setDefault().to(DefaultGlueColumnStatisticsProviderFactory.class).in(Scopes.SINGLETON);
+
+        createExecutor(ForGlueHiveMetastore.class, "hive-glue-partitions-%s", GlueHiveMetastoreConfig.class, GlueHiveMetastoreConfig::getGetPartitionThreads);
```

Comment:
no need to paramerize with `GlueHiveMetastoreConfig.class`

Reply:
done